index.js

```diff
@@ -52,12 +52,13 @@ function start(pi, cb) {
         restartOk: pi.restartOk,
         cb: cb,
     }
-
+    
     if(!pi.restartAt) pi.restartAt = [100,500,1000,30*1000,60*1000,5*60*1000,15*60*1000]
     if(!pi.restartOk) pi.restartOk = 30 * 60 * 1000
 
     get_script_1(pi, (script) => {
         pi._script = script
+        pi = asarCwdScript(pi)
```

Comment:
Design comments:
1. asarCwdScript doesn't mean anything. Change to 'fixAsarIssue'
2. The function returns `pi` but also modifies `pi`. It should do one or the other (in formal terminology we would say it should handle "by reference" or "by value"). Do not return pi (handle by reference).

Reply:
sure Charles, I will make changes